src/value.rs

```diff
@@ -69,7 +71,46 @@ impl Value {
 
     /// Serializes a `Value` to a byte stream as an XML encoded plist.
     pub fn to_writer_xml<W: Write>(&self, writer: W) -> Result<(), Error> {
-        let mut writer = XmlWriter::new(writer);
+        self.to_writer_xml_with_options(writer, WriteOptions::default())
+    }
+
+    /// Serializes a `Value` to a stream, using custom [`WriteOptions`].
+    ///
+    /// If you need to serialize to a file, you must acquire an appropriate
+    /// `Write` handle yourself.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::io::{BufWriter, Write};
+    /// use std::fs::File;
+    /// use plist::{Dictionary, Value};
+    /// use plist::stream::{WriteOptions, XmlWriter};
+    ///
+    /// let value: Value = Dictionary::new().into();
+    /// // .. add some keys & values
+    /// let mut file = File::create("com.example.myPlist.plist").unwrap();
+    /// let options = WriteOptions::default().indent_string("  ");
+    /// value.to_writer_xml_with_options(BufWriter::new(&mut file), options).unwrap();
+    /// file.sync_all().unwrap();
+    /// ```
+    #[cfg(feature = "enable_unstable_features_that_may_break_with_minor_version_bumps")]
```

Comment:
This can go straight to being stable. `enable_unstable_features_that_may_break_with_minor_version_bumps` is really only there to cover the `stream` module.

Reply:
Should `XmlWriteOptions` be public API, then? currently it's in the `stream` module, but I could reexport it somewhere?

Reply:
Yes, I didn't realise that `XmlWriteOptions` wasn't in the crate root.